conda_index/index/sqlitecache.py

```diff
@@ -224,12 +240,21 @@ def _extract_to_cache(
             ).fetchone()
 
             if cached_row and not second_try:
-                log.info("Found %s in cache" % fn)
+                # log in caller?
+                log.debug("Found %s in cache" % fn)
```

Comment:
possibly recompute shasums here and fall back to extraction if they do not match